src/reference/asciidoc/aggregator.adoc

```diff
@@ -812,6 +812,13 @@ The callback has direct access to the store and the message group so that it can
 The `MessageGroupStore` maintains a list of these callbacks, which it applies, on demand, to all messages whose timestamps are earlier than a time supplied as a parameter (see the `registerMessageGroupExpiryCallback(..)` and `expireMessageGroups(..)` methods, described earlier).
 For more detail, see <<reaper>>.
 
+IMPORTANT: It is important do not use the same `MessageGroupStore` instance store in different aggregator components, when you are intended to rely on the `expireMessageGroups` functionality.
+Every `AbstractCorrelatingMessageHandler` registers its own `MessageGroupCallback` based on the `forceComplete()` callback.
+This way each group for expiration may be completed or discarded by wrong aggregator.
```

Comment:
If the same store is used in multiple components, each expiring group may be ... by **the** wrong ...